zerver/openapi/zulip.yaml

```diff
@@ -15541,27 +15541,29 @@ paths:
                     Whether the data export should be public, full with consent,
                     or full without consent.
 
-                    - 1 = Public data export.
-                    - 2 = Full with consent data export, which includes private data
+                    - `public` = Public data export.
+                    - `full_with_consent` = Full with consent data export, which includes private data
                       for users who have granted consent.
-                    - 3 = Full without consent data export, which includes private data for
+                    - `full_without_consent` = Full without consent data export, which includes private data for
                       all users. This option requires the organization to have
                       the `owner_full_content_access` feature enabled.
 
-                    If not specified, defaults to 1.
+                    If not specified, defaults to `public`.
 
-                    **Changes**: Zulip 12.0 (feature level ZF-712cf1) added the value 3
-                    option for full data export without member consent.
+                    **Changes**: Zulip 12.0 (feature level ZF-712cf1) changed the type of
+                    this field from int to string with `1` being replaced by `public` and
+                    `2` being replaced by `full_with_consent`. We also added another option
+                    `full_without_consent` for full exports without member consent.
```

Comment:
Note on style for these changes entries: We never say "We" here, we're always just describing in passive voice what the change was.